pkg/actors/config/options.go

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package internal
```

Comment:
This file should not be in `config`. Maybe more in a package like `interfaces`, together with the placement interface.

Reply:
No problem. Will move.